style.css

```diff
@@ -0,0 +1,164 @@
+html,
+body {
+  font-family: sans-serif;
+  margin: 0;
+  width: 100%;
+  font-size: 0.9em;
+}
+
+.book {
+  padding: 5px;
+  margin-bottom: 20px;
+  margin-top: 20px;
+}
+
+.author {
+  padding: 5px;
+  margin-bottom: 10px;
+}
+
+.menu ul {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  color: white;
+  font-weight: 300;
+  letter-spacing: 2px;
+  text-decoration: none;
+  background: rgb(238, 13, 13);
+  padding: 20px 5px;
+  display: inline-block;
+  width: 100%;
+  text-align: center;
+}
+
+a:hover {
+  background: rgba(238, 13, 13, 0.75);
+}
+
+.navigation ul {
+  border: 1px solid rgb(212, 20, 20);
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+  flex-grow: 1;
+}
+
+main {
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 0 3em 1.5em;
+  display: flex;
+}
+
+.navigation li {
+  flex: 3;
+  list-style: none;
+}
+
+.navigation .links {
+  flex: 1;
+}
+
+.navigation .awesome-link a {
+  text-align: left;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.book-list {
+  display: flex;
+  justify-content: space-between;
+}
+
+.container > div:nth-child(even) {
+  background-color: lightgreen;
+  padding-top: 4px;
+  margin-top: 5px;
+  padding-bottom: 4px;
+}
+
+#date-time {
+  font-weight: bold;
+  text-align: right;
+  padding: 4px;
+}
+
+.box {
+  transition: 1s ease-in-out;
+  -webkit-transition: 1s ease-in-out;
+  width: 100%;
+  display: none;
+}
+
+#list h1 {
+  font-size: 20px;
+  display: flex;
+  justify-content: center;
+}
+
+#form {
+  width: 100%;
+  margin-left: 35%;
+}
+
+#form h1 {
+  font-size: 20px;
+  margin-left: 5%;
+}
+
+#form input {
+  width: 50%;
+}
+
+button {
+  font-size: 14px;
+  box-shadow: 3px 2px 1px 1px black;
+}
+
+#form button {
+  position: absolute;
+  right: 33%;
+}
+
+#contact {
+  margin-left: 30%;
+  margin-top: 7%;
+}
+
+#contact h1 {
+  font-size: 20px;
+  margin-left: 10%;
+}
+
+#contact p {
+  font-weight: 400;
+  font-size: 15px;
+}
+
+#contact ul {
+  margin-top: 20px;
+  font-size: 15px;
+}
+
+footer {
+  margin-left: 5%;
+  padding: 8px;
+  border-radius: 3px;
+  height: 30px;
+  width: 90%;
+  background: rgb(226, 223, 223);
+  margin-top: 14%;
+  text-align: center;
+}
```

Comment:
- [ ] It would be nice to position the Footer at the bottom of the page. Footers are usually positioned at the bottom of the page and they make the website more beautiful when positioned that way. Also, positioning your Footer at the bottom will make your design look identical to the wireframe provided. 👍  

| Your Design | Wireframe Provided |
| ---------------------- | -------------- |
|![image](https://user-images.githubusercontent.com/77106962/236045880-ad422e98-1101-4ac6-baa8-77537b8c0471.png) | ![image](https://user-images.githubusercontent.com/77106962/236046122-73bc2212-cea9-4d24-bcb3-d48eda518309.png) |